src/mapeo-manager.js

```diff
@@ -0,0 +1,194 @@
+import { randomBytes } from 'crypto'
+import path from 'path'
+import { serialize, deserialize } from 'v8'
```

Comment:
this seems to be the preferred way of (de)serializing a buffer for on-disk persistence. https://nodejs.org/docs/latest-v18.x/api/v8.html#v8serializevalue

This avoids json stringifying the encryption keys object and then creating a buffer from it, which changes the actual structure of the object for fields with buffer values i.e. 

```js
JSON.stringify({ foo: Buffer.from('foo') })
``` 

turns into

```json
{"foo":{"type":"Buffer","data":[102,111,111]}}
```

which requires extra work when reading from the db and deserializing

Reply:
This makes me think that we should actually export a method from @mapeo/crypto specifically for encrypt/decrypt of this e.g. `decryptProjectKeys()`. I think this is ok for now, but can you open an issue to address this.

Reply:
done! https://github.com/digidem/mapeo-crypto/issues/10

Reply:
addressed via https://github.com/digidem/mapeo-core-next/pull/199/commits/6d14bacb6f178af5102c8d855efffbb0d3b4c085